packages/@aws-cdk/region-info/build-tools/static-data-facts/promised-region-info.ts

```diff
@@ -0,0 +1,92 @@
+import { IRegionInfo } from '../../lib/iregion-info';
+import { AwsServiceNames } from './index';
+import { servicePrincipal } from './service-principals';
+
+export class PromisedRegionInfo {
+  public readonly partition: Promise<string>;
```

Comment:
Why are these promises? So you can read them from a file?

Reply:
So I don't have to care in which order they get a value.